editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -211,9 +212,19 @@ void VisualShaderGraphPlugin::set_uniform_name(VisualShader::Type p_type, int p_
 }
 
 void VisualShaderGraphPlugin::update_curve(int p_node_id) {
-	if (links.has(p_node_id) && links[p_node_id].curve_editor) {
+	if (links.has(p_node_id) && links[p_node_id].curve_editors[0]) {
```

Comment:
Will editors ever be 0?

Reply:
What do you mean? This is a check to prevent crashes. This array filled with `nullptr` by default (see VisualShaderGraphPlugin::register_link).